lib/file-source/src/checkpointer.rs

```diff
@@ -62,7 +106,7 @@ impl Checkpointer {
             }
             'i' => {
                 let (dev, ino, pos) =
-                    scan_fmt!(file_name, "i{x}.{y}.{}", [hex u64], [hex u64], FilePosition)
+                    scan_fmt!(file_name, "i{x}.{x}.{}", [hex u64], [hex u64], FilePosition)
```

Comment:
Is this intentional?

Reply:
Yes, this is the bug fix mentioned in the description. I can pull this commit out to its own PR if that'd be helpful, as I think it needs to land before 0.11 /cc @binarylogic

Reply:
I don't mind it here.